src/main/java/org/mitre/synthea/engine/Module.java

```diff
@@ -423,7 +440,7 @@ public synchronized Module get() {
       if (fault != null) {
         throw new RuntimeException(fault);
       }
-      return module;
+      return module.clone();
```

Comment:
Have you looked at how much this affects memory usage? The whole set of modules as raw text is 2 MB so broken out into full objects across multiple patients that might be a lot. If the real issue was the race condition in PediatricGrowthTrajectory and this increases memory usage by a lot, we may want to revert this piece.

Reply:
<img width="552" alt="image" src="https://user-images.githubusercontent.com/1054765/88101068-7601f380-cb6b-11ea-902b-9926d45c5f41.png">

This is a run of 1000 patients. My max memory is set to 8 GB.

Reply:
<img width="552" alt="image" src="https://user-images.githubusercontent.com/1054765/88101374-e01a9880-cb6b-11ea-976b-35cc4afe5add.png">

Same run of 1000 patients from the latest `master`

Reply:
Thanks, doesn't seem like there's a meaningful difference